### DISCUSSION TITLE
v1.8 release

### DISCUSSION THREAD

**Discussion by samuelcolvin** [pydantic/pydantic#2355] [General]
I want to make a v1.8 release as soon as possible.

I've merged all PRs which were ready for review. I'm now going to go through all the other PRs and see if anything can be merged or closed.

If you have anything else you think should go into v1.8, please comment here.

@PrettyWood @StephenBrown2 @layday @MrMrRobat

Should this be a discussion or an issue? It makes more sense as a discussion since it's no "an issue" with the code, but it's annoying that I don't seem to be able to "close" or "archive" it.

**Comment by Bobronium** [comment]
Thanks for tagging! 

I'd love to see #2356 in 1.8
Outdated part:
<details>though there's no opened PR yet, I'm planning to open one today. I managed to implement this feature already, now it's a matter of writing tests and docs. 
Also I can open PR now, so you could glance at current implementation to decide whether it can go into 1.8 or need some polishing.
</details>


As of «discussion/issue» question, I don't have a preference here at all :)

**Comment by PrettyWood** [comment]
I feel like for releases you could open a "meta" issue for now as you may indeed want to close/archive it. In the future it may be possible to do the same with discussions. They are actively adding features to it and it's probably something that they will implement.

---------

As for PRs I guess v1.8 is already fat enough but I guess the work to [support properly `Literal` in JSON schema](https://github.com/samuelcolvin/pydantic/pull/2348) could be useful
The work on `Union` with [discriminated union](https://github.com/samuelcolvin/pydantic/pull/2336) and [smart union](https://github.com/samuelcolvin/pydantic/pull/2092#issuecomment-763833122) can definitely wait for v1.9. Would love your feedback on these ones though whenever you have time :)

**Comment by layday** [comment]
Thanks for the ping.  Would like to see #1881 in 1.8 if we hear back from the author.

**Comment by samuelcolvin** [comment]
I've merged everything I can. I'll release v1.8 soon unless anyone points out a blocker.

Thanks for everyone's work.

---

Regarding discussions vs. issues. This should have been an issue.

**Reply by Bobronium** [reply]
Thank you!
